src/integrations/Appcues/browser.js

```diff
@@ -21,7 +23,14 @@ class Appcues {
 
   init() {
     logger.debug('===in init Appcues===');
-    ScriptLoader('appcues-id', `https://fast.appcues.com/${this.accountId}.js`);
+
+    let url = `https://fast.appcues.com/${this.accountId}.js`;
+    if (isDefinedAndNotNullAndNotEmpty(this.proxyUrl) && typeof this.proxyUrl === 'string') {
+      url = this.proxyUrl.includes(`${this.accountId}.js`)
+        ? this.proxyUrl
+        : `${this.proxyUrl}/${this.accountId}.js`;
```

Comment:
You need to ensure that there are no double `/` as a result of this concatenation.

Reply:
updated